sdk-python/prisma/schema.prisma

```diff
@@ -0,0 +1,19 @@
+datasource client {
+  provider = "sqlite"
+  url      = "file:dev.db"
+}
+
+generator client {
+  provider = "prisma-client-py"
+  interface = "asyncio"
+  recursive_type_depth = -1
+}
+
+model Log {
```

Comment:
I suppose this is for local development and wouldn't actually be included in the sdk?

We now have this `schema.prisma` as well as the `front/prisma/schema.prisma` - I wonder if we can reduce duplication